codegen.yml

```
@@ -1,5 +1,5 @@
 overwrite: true
-schema: 'https://dev.v1.api.tabetalt.no/'
+schema: http://localhost:3000/
```

Under normal conditions, we probably want to use the external API.

```suggestion
schema: https://dev.v1.api.tabetalt.no/
```

What do you think?

---

Fixed